Blog post about sensible defaults.

amjith opened the pull request:
@tsroten Whaddya think?

j-bennet commented:
Good stuff @amjith. You might want to mention your recent `casing=auto` also. :)

I think there's one thing we didn't quite get with our sensible defaults (not mentioned in the article): decimal and float format specifiers. We left them blank by default, but it might have been a better solution to extract that information from user's locale settings.

tsroten commented on content/sensible-defaults.md:
```diff
@@ -0,0 +1,76 @@
+for different databases. 
+
+The feature described in the blogpost talks about a cool feature to store the
+history of a sessio scoped to a database. This is an awesome feature that we
```
Needs an `n` on `sessio`

:smile:

tsroten commented:
@amjith This looks great! 👍 I'm happy with it as is, but I also think @j-bennet's suggestion is a good one. Maybe an "Additional Features" section at the bottom to talk about a couple other configs that Craig didn't mention?

Another possibility would be the sensible defaults for syntax colors, which is also overridable.

amjith commented:
Fixed the typo. Thank you!


Regarding the expansion; I think it is best to keep the post short. I've changed the conclusion to say that we have a lot more config options and provided a link to the config file.

My hope is that the blogpost should make people intrigued and the motivated ones will checkout the rest of the config file which will further motivate them to try pgcli.

If we see good reception for this post I'll be happy to write a follow-up post that goes into a little bit more depth.

amjith commented:
BTW, I don't know if you guys already saw this, but since pgcli.com uses netlify we can generate previews and the link to the preview is posted here: 

<img width="816" alt="blog_post_about_sensible_defaults__by_amjith_ _pull_request__16_ _dbcli_pgcli_com" src="https://user-images.githubusercontent.com/49260/28270588-d998387e-6aba-11e7-9568-9fe27d916e89.png">

We should do the same for mycli.net soon.

j-bennet commented:
What the what! I also didn't notice the preview! This is amazing!

@amjith I'd only add the `casing=auto` to your post then. Casing seems to be important to people. :)

amjith commented:
I'll add the casing option and push a new commit.
